modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7373,19 +7528,60 @@ public void testGetScopeScopeKeyObjectEntriesPage() throws Exception {
 				HTTPTestUtil.invokeToJSONObject(
 					null,
 					_getEndpoint(
-						RandomTestUtil.randomLong(),
+						null, RandomTestUtil.randomLong(),
 						_siteScopedObjectDefinition1),
 					Http.Method.GET));
 		}
 
+		// Using groupExternalReferenceCode
```

Comment:
I would remove this comment.
I would change `// Site scope` to `// Site scope external reference code`